### DISCUSSION TITLE
How to get tag associated with commit

### DISCUSSION THREAD

**Discussion by sachin-shitole-paccar** [rfennell/AzurePipelines#1880] [Q&A]
We are generating release notes using XplatGenerateReleaseNotes extension. The release notes are generated at build stage on the basis of _overrideBuildReleaseId_ parameter. We want to enrich release notes with the tag associated with each commit which is done after the provided buildId.

FYI: GitVersion is being used to generate build version.

We do get all the PRs and commits while generating release notes but couldn't find a way to get tag associated with each commit.

Does anyone have any idea how to retrieve it?

**Comment by rfennell** [comment]
The task only support Azure DevOps pipeline tags, not Git tags, so what you ask is not possible
